ios/brave-ios/Sources/BraveShared/BraveURLs.swift

```diff
@@ -82,6 +82,10 @@ extension URL {
       string:
         "https://support.brave.com/hc/en-us/articles/32389914657549-What-is-the-Brave-VPN-Kill-Switch"
     )!
+    public static let newTabTakeoverLearnMoreLinkUrl = URL(
```

Comment:
oh we have to redefine all these strings for ios? :cry:

Reply:
I didn't find a proper way of sharing such strings between platforms. 
cc @brave/ios